src/widget.ts

```diff
@@ -65,6 +65,7 @@ abstract class ReteIOModel extends DOMWidgetModel {
       ...super.defaults(),
       key: undefined,
       title: undefined,
+      multi_connection: undefined,
```

Comment:
```suggestion
      multi_connection: false,
```

Let's set this to false by default.